wan/README.md

```diff
@@ -0,0 +1,125 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Geode WAN replication example
```

Comment:
How much previous reading is someone supposed to have done about geode WAN? It seems like we're using terms like gateway sender and receiver without explaining what they mean. Maybe the README should actually describe what a receiver and a sender do, and why this is example is bidirectional. 

Also, serial sender's aren't just used for replicated regions, they are also for ensuring the order of updates on the remote side. Maybe the readme should just mention it's using parallel senders and point to the docs for serial vs. parallel.

Reply:
I have updated the README.md file with a bit more explanation of gateway senders and receivers and added a specific link to the Geode documentation about WAN configuration. Please let me know if you think this is sufficient.